### ISSUE TITLE
NTR: Modifier: Elbow predominance (Extremities, joints & nails)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7973]
New term request Modifier: Elbow predominance (sundel_140516150235): Appendicular  UMLS: 0013769

**Comment by pnrobinson** [comment]
@MickeySegal -- same question as for predominent heal.
Or is this referring to a joint abnormality that is most severe in the elbow?

**Comment by MickeySegal** [comment]
The finding is labeled as a Modifier to indicate that it gives further detail about another finding. There are similar findings for unilateral, distal, proximal, elbow, shoulder, hip and so forth.

**Comment by pnrobinson** [comment]
@MickeySegal 
The HPO has a hierarchy of terms like this that refer to specific locations (without including the concept predominance)
[Spatial pattern HP:0012836](https://hpo.jax.org/app/browse/term/HP:0012836)
Of course there is a subtle distinction between say "Distal" and "Predominantly distal", but in practice in medicine, almost everything is actually "predominant X" and maybe we can add these concepts without using "predominent" in the label for consistency? The problem is that if you label the same patiebnt with say "knee predoiment" and "elbow predominant" X, this is logically untrue because neither predominants over the other...

**Comment by MickeySegal** [comment]
The term "more severe" may be more clear than "predominance", but it is not wrong to say that a person's arthritis is predominately in the elbow.  It may be good for both of us to include both terms.

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000136
- label: Elbow localization
- parents: [Localized (HP:0012838)](https://hpo.jax.org/app/browse/term/HP:0012838)
- pmid: 
- def: Applies to an abnormality that is situated in (or predominant in) the elbow.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7973
- synonyms:
